dev-server/documents/html/sidebar-external-container.mustache

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Annotatable iframe test</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+      .container {
+        display: flex;
+      }
+      #my-sidebar {
+        flex: 0 0 20%;
+      }
+      iframe {
+        flex: 1;
+        height: 400px;
+        resize: both;
```

Comment:
What is the purpose of `resize`? In my testing it seems that the content iframe is resizable but only in the vertical direction.